pkg/common/cns-lib/volume/manager.go

```diff
@@ -213,6 +218,8 @@ type defaultManager struct {
 	operationStore                 cnsvolumeoperationrequest.VolumeOperationRequest
 	idempotencyHandlingEnabled     bool
 	multivCenterTopologyDeployment bool
+	improvedTasksListViewEnabled bool
+	taskViewManager              TaskManagerIf
```

Comment:
backend API we are using is Listview, why we are naming it as taskviewManager?

Reply:
Fixed.